quick-start.md

```diff
@@ -56,7 +71,7 @@ The downside to switching to the 'overlay2' storage driver after creating and wo
 git clone https://github.com/RadeonOpenCompute/ROCm-docker
 cd ROCm-docker
 sudo docker build -t rocm/rocm-terminal rocm-terminal
-sudo docker run -it --rm --device="/dev/kfd" rocm/rocm-terminal
+sudo docker run -it --network=host --device=/dev/kfd --device=/dev/dri --group-add video rocm/rocm-terminal
```

Comment:
Are you sure this is complete?  Don't you need the `--privileged` flag to access the `/dev/dri` device?

Reply:
or may be being part of the video group is sufficient?  (I am not up-to-date on the latest.)

Reply:
@parallelo , could you help for the question here?
The above command has been working fine for internal development.

Reply:
If it works then it's fine with me.  I just recall some earlier complaint/problem that was fixed with using --privileged flag.  If being part of the video group is a better fix I am all for it.

Reply:
Yes.  Good to drop `--privileged`.  Also can drop `--network=host` (was using that trick specifically for TensorBoard on a headless server.  Below is probably the minimal command:   
`docker run -it --device=/dev/kfd --device=/dev/dri --group-add video rocm/rocm-terminal`

Reply:
Quick test looks good:
```
$ docker run -it --device=/dev/kfd --device=/dev/dri --group-add=video rocm/rocm-terminal

rocm-user@fa0de34a5461:~$ cd /opt/rocm/hip/samples/0_Intro/square/

rocm-user@fa0de34a5461:/opt/rocm/hip/samples/0_Intro/square$ sudo make
/opt/rocm/hip/bin/hipify-perl square.cu > square.cpp
/opt/rocm/hip/bin/hipcc  square.cpp -o square.out
./square.out

rocm-user@fa0de34a5461:/opt/rocm/hip/samples/0_Intro/square$ ./square.out
info: running on device Device 6860
info: allocate host mem (  7.63 MB)
info: allocate device mem (  7.63 MB)
info: copy Host2Device
info: launch 'vector_square' kernel
info: copy Device2Host
info: check result
PASSED!
```